CIP-0068/README.md

```diff
@@ -246,6 +246,92 @@ We want to bring the metadata of the FT `d5e6bf0500378d4f0da4e8dde6becec7621cd8c
 3. Reference the output in the transaction. (off-chain)
 4. Verify validity of datum of the referenced output by checking if policy ID of `reference NFT` and `user token` and their asset names without the `asset_name_label` prefix match. (on-chain)
 
+## 444 SFT Standard
+
+The third introduced standard is the `444` Semi-FT standard with the registered `asset_name_label` prefix value
+
+| asset_name_label            | class        | description                                                          |
+| --------------------------- | ------------ | -------------------------------------------------------------------- |
+| 444                         | SFT          | SFT hold by the user's wallet making use of the union of CIP-0025 inner structure AND the Cardano foundation off-chain registry inner structure |
+
+Semi-Fungible tokens don't fit cleanly into the other two FT/NFT classes of tokens and thus need their own standard. An example of a SFT would be a fractionalized NFT. The single reference NFT `(100)` represents the NFT itself, and the many `(444)` tokens represent the fractionalized shares. Minting 100 tokens and setting decimals to 2 would represent a single NFT that is split into 100 fractions.
+
+### Class
+
+The `user token` is an SFT (semi-fungible token).
+
+### Pattern
+
+The `user token` and `reference NFT` **must** have an identical name, preceded by the `asset_name_label` prefix.
+
+Example:\
+`user token`: `(444)Test123`\
+`reference NFT`: `(100)Test123`
+
+### Metadata
+
+This is a low-level representation of the metadata, following closely the structure of CIP-0025 combined with the Cardano foundation off-chain metadata registry. All UTF-8 encoded keys and values need to be converted into their respective byte's representation when creating the datum on-chain.
+
+```
+files_details = 
+  {
+    ? name : bounded_bytes, ; UTF-8
+    mediaType : bounded_bytes, ; UTF-8
+    src : bounded_bytes ; UTF-8
+    ; ... Additional properties are allowed
+  }
+
+metadata = 
+  {
+    name : bounded_bytes, ; UTF-8
+    description : bounded_bytes, ; UTF-8
+    image : bounded_bytes, ; UTF-8
+    ? mediaType : bounded_bytes, ; UTF-8
+    ? ticker: bounded_bytes, ; UTF-8
+    ? url: bounded_bytes, ; UTF-8
+    ? logo: uri,
```

Comment:
`logo` and `image` have the same purpose. I see you are merging the fields from the `222` and `333` standard. It's probably enough to just have the fields from the `222` label with the `decimals` field included.

Reply:
Modified so it matches 222 with decimals field added.